### DISCUSSION TITLE
Unterstützungsanfrage für AMD-Grafikkarten-Kompatibilität mit Ihrer Software "Fooocus"

### DISCUSSION THREAD

**Discussion by EngelNight1** [lllyasviel/Fooocus#2847] [Q&A]
Sehr geehrte Damen und Herren,

ich wende mich an Sie, da ich bei der Nutzung Ihrer Software "Fooocus" auf technische Schwierigkeiten gestoßen bin. Die Software scheint eine NVIDIA-Grafikkarte für die Ausführung zu erfordern, jedoch verfügt mein System über eine AMD-Grafikkarte, was zu Kompatibilitätsproblemen führt.

Hier die Details meiner Systemkonfiguration:

Prozessor: AMD Ryzen 9 6900HX mit Radeon Graphics, 8 Kerne, 16 Threads, Basis-/Boost-Geschwindigkeit von 4,737 GHz
Grafikkarte: AMD Radeon RX 6600
Compute Units: 28
Ray Accelerators: 28
Infinity Cache: 32 MB
Grafikspeicher: 8 GB GDDR6
Game Clock: Bis zu 2044 MHz
Boost-Frequenz: Bis zu 2491 MHz
Speicherbandbreite: Bis zu 224 GB/s
Angesichts dieser Konfiguration möchte ich Sie bitten zu prüfen, ob es möglich ist, Ihre Software für die Nutzung mit AMD-Hardware anzupassen oder ob bereits eine kompatible Version existiert. Die AMD Radeon RX 6600 ist für anspruchsvolles 1080p-Gaming optimiert und unterstützt moderne Grafiktechnologien, was sie zu einer leistungsstarken Option für multimediale Anwendungen macht.

Ich wäre Ihnen sehr dankbar, wenn Sie mir mitteilen könnten, ob und wie ich Ihre Software unter diesen Voraussetzungen verwenden kann. Für weitere Rückfragen stehe ich Ihnen gerne zur Verfügung.

Mit freundlichen Grüßen,

**Comment by ILikeToasters** [comment]
AMD is supported and you would need to follow the instructions on the main page -  AMD wird unterstützt und Sie müssen den Anweisungen auf der Hauptseite folgen: https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#linux-amd-gpus
